config/jobs/kubernetes-sigs/cluster-api-provider-digitalocean/cluster-api-provider-digitalocean-periodics.yaml

```diff
@@ -1,4 +1,34 @@
 periodics:
+- name: periodic-cluster-api-provider-digitalocean-janitor
+  decorate: true
+  decoration_config:
+    timeout: 1h
+  interval: 12h
+  labels:
+    preset-dind-enabled: "true"
+    preset-kind-volume-mounts: "true"
+    preset-do-credential: "true"
+  extra_refs:
+    - org: kubernetes-sigs
+      repo: cluster-api-provider-digitalocean
+      base_ref: master
+      path_alias: "sigs.k8s.io/cluster-api-provider-digitalocean"
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/kubekins-e2e:v20201203-4778e22-master
+      command:
+        - "runner.sh"
+        - "./scripts/ci-janitor.sh"
+      securityContext:
+        privileged: true
```

Comment:
Related to the comment above, do we need privileged securityContext in this case?